project/plugins.sbt

```diff
@@ -1,4 +1,3 @@
-
-addSbtPlugin("io.verizon.build" % "sbt-rig" % "1.1.20")
+addSbtPlugin("io.verizon.build" % "sbt-rig" % "1.1.22")
```

Comment:
This one needs to be `1.2.25` as this is the freshly built version available on central